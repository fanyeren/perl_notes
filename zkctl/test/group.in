#!/bin/bash
source $(dirname $0)/common

node_clear group

./zkctl -w group add --group-id=group_1 --data='{"info":"hello worlp"}'
echo "add result $?"
./zkctl -w group get-meta-all --group-id=group_1
echo "getmeta-all result=$?"
./zkctl -w group get-meta-attr --group-id=group_1 --key=info
echo "getmeta-attr result=$?"
./zkctl -w group del --group-id=group_1
echo "del group result=$?"

