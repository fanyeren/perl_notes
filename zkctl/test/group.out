add result 0
{"info":"hello worlp"}
getmeta-all result=0
hello worlp
getmeta-attr result=0
del group result=0
